README.rst

```diff
@@ -53,6 +53,12 @@ Then, place your keystore file in ``$(tutor config printroot)/env/plugins/androi
 
     tutor local start -d android-app
 
+Making courses visible in app
```

Comment:
Let's not put this section smack in the middle of the Android app build/release instructions. You can remove the title and move the instructions line 37, right in the "Usage" section.